src/calendar/interfaces/calendar.interface.ts

```diff
@@ -1,3 +1,5 @@
+import { Author } from '../../auth/getAuthorFromHeaders.decorator';
```

Comment:
For a future idea / PR, what do you think about using path alias, something like

```
import { Author } from '@auth/getAuthorFromHeaders.decorator';
```

https://docs.nestjs.com/cli/libraries